tft_bot/helpers/system_helpers.py

```diff
@@ -1,7 +1,7 @@
 """A collection of system-level helpers"""
-import http.client as httplib
```

Comment:
Any major benefit to the switch to using a socket connection instead of a regular HEAD request? 🤔

Reply:
This I actually have a reason for. A host and port connection through a socket to the DNS service is more reliable than an HTTP HEAD request. Since HTTP is on the application layer, it resolves the DNS first (even if we use a DNS service IP). Sockets are on the transport layer, which does not do this and really only checks if we can connect to the outside.